### PR TITLE
Version 3.59.0

### PR DESCRIPTION
**Pull request by bethbrains**
Updated HISTORY.rst and version.txt. Fixed some typos in HISTORY.rst.

# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 History
 =======
-3.58.0 (2022-04-16)
+3.59.0 (2022-05-26)
+-------------------
+
+Changes:
+
+- Setting ``repo_url`` in your project config is no longer required to run ``metadeploy_publish``. (#3200)
+- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the robot framework Browser library which is based on Playwright. (#3194)
```
Comment (davidmreed) [inline HISTORY.rst:10]:
```suggestion
- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the Robot Framework ``Browser`` library, which is based on Playwright. (#3194)
```

**Review by davidmreed — APPROVED** [review]

File: HISTORY.rst (hunk 1 above)
Comment (boakley) [reply HISTORY.rst]: We might want to add "For more information see the section "Playwright Technology Preview" in the public documentation.

File: HISTORY.rst [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 History
 =======
-3.58.0 (2022-04-16)
+3.59.0 (2022-05-26)
+-------------------
+
+Changes:
+
+- Setting ``repo_url`` in your project config is no longer required to run ``metadeploy_publish``. (#3200)
+- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the Robot Framework ``Browser`` library, which is based on Playwright. (#3194)
```
Comment (boakley) [inline HISTORY.rst:10]: We might want to add "For more information see the section "Playwright Technology Preview" in the public documentation.

**Review by boakley — DISMISSED** [review]

File: HISTORY.rst (hunk 2 above)
Comment (Br4nd0R) [reply HISTORY.rst:10]:
```suggestion
- Adds new commands ``cci robot install_playwright`` and ``cci robot uninstall_playwright`` for installing and uninstalling experimental support for the Robot Framework ``Browser`` library, which is based on Playwright. For more information see the section "Playwright Technology Preview" in the public documentation. (#3194)
```

**Review by Br4nd0R — APPROVED** [review]

**Pull request merged**